tests/cache.py

```diff
@@ -11,7 +11,7 @@
 
 
 britive = Britive()  # source details from environment variables
-profiles_v1 = britive.feature_flags['profile-v1']
+profiles_v1 = False
```

Comment:
I have to check, but should we remove these 4 variables?

Reply:
yes, lets remove all references to profile v1 and v2 in the test code.